client/src/pages/open-source-skolan/[slug].js

```diff
@@ -0,0 +1,177 @@
+          openGraph={{
+            title: lesson?.metaTags?.title ?? lesson.title,
+            description:
+              lesson?.metaTags?.description ?? lesson?.imageCard?.description,
+            images: [
+              {
+                url: urlFor(
+                  lesson?.metaTags?.imageWithAlt?.asset._ref ??
+                    lesson?.preview?.imageCard?.image.asset._ref
+                ),
+              },
+            ],
+            site_name: 'Iteam',
+          }}
+          twitter={{
+            title: lesson?.metaTags?.title ?? lesson.title,
+            description:
+              lesson?.metaTags?.description ?? lesson?.imageCard?.description,
+            image: urlFor(
+              lesson?.metaTags?.imageWithAlt?.asset._ref ??
+                lesson?.preview?.imageCard?.image.asset._ref
+            ),
+            handle: '@iteam1337',
+            site: '@iteam1337',
+            cardType: 'summary_large_image',
+          }}
+        />
+      )}
+      <PageWrapper footerDark>
+        {preview && <ExitPreviewLink />}
+        <Section className="pb-0">
+          <div className="pt-5"></div>
+          <Container>
+            <Row className="justify-content-center text-center">
+              <Col lg="12">
+                <Title variant="hero">{lesson?.title && lesson.title}</Title>
+                <Box className="d-flex justify-content-center">
+                  <Text>{lesson?.date && lesson.date}</Text>
+                </Box>
+              </Col>
+            </Row>
+            {lesson?.youtubeId && (
+              <Row>
+                <Col>
+                  <Box mt={96} mb={32}>
+                    <VideoContainer>
+                      <VideoIframe
+                        src={`https://www.youtube.com/embed/${lesson?.youtubeId}?autoplay=1`}
+                        title={lesson?.metaTags?.title ?? lesson.title}
+                        frameborder="0"
+                        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                        autoplay
+                        allowfullscreen />
+                    </VideoContainer>
+                  </Box>
+                  <Box>
+                    <Text><Span fontWeight='bold'>Open Source skolan:</Span> {lesson?.imageCard?.description && lesson.imageCard.description}</Text>
+                  </Box>
+                </Col>
+              </Row>
+            )}
+          </Container>
+        </Section>
+        {data?.lessons && (
+          <>
+            <Section className="pb-0">
+              <div className="pt-5"></div>
+              <Container>
+                <Row className="justify-content-center text-center">
+                  <Col lg="12">
+                    <Title>Se fler lektioner i Open Source skolan</Title>
+                  </Col>
+                </Row>
+              </Container>
+            </Section>
+            <BlogList posts={data.lessons} />
```

Comment:
Design-wise and on desktop (looks good on mobile), I think it would look better with less top padding (half?) for this section.